src/collections/components/AddLabelForm/AddLabelForm.validation.tsx

```diff
@@ -0,0 +1,9 @@
+import * as yup from 'yup';
```

Comment:
Minimal validation for now to move this PR. More validation to come in a separate PR after establishing what else needs to be validated.

Reply:
I think it's a good idea to implement validation right from the start. We know that labels should be slug-compatible, so reusing validation rules in place for Collection/CollectionStory/CollectionAuthor slugs should be fine here. Also need to enforce this on the backend - I assume you have this under control already.

Reply:
@nina-py yep, that is a good point however we still need to collect requirements like min/max size and if we allow non-slug style. Also, for example if it is a `region` kind of label, we might want to validate it starts with `region`. We don't have all the requirements for now and set aside for another ticket.